src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeStore.targets

```diff
@@ -211,6 +220,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       <SkipUnchangedFiles Condition="'$(SkipUnchangedFiles)' == ''">true</SkipUnchangedFiles>
     </PropertyGroup>
 
+   <NETSdkError Condition="'2.0' > '$(_TargetFrameworkVersionWithoutV)'"
+                 ResourceName="NU1008"
+                 FormatArguments=".NET Core $(_TargetFrameworkVersionWithoutV)"/>
```

Comment:
I meant `$(TargetFrameworkMoniker)` above.  Not Identifier, since Identifier doesn't have the version.

---
In reply to: [120181618](https://github.com/dotnet/sdk/pull/1297#discussion_r120181618) [](ancestors = 120181618)